src/naive/internals.rs

```diff
@@ -268,7 +268,7 @@ const OL_TO_MDL: &[u8; MAX_OL as usize + 1] = &[
 /// which is an index to the `OL_TO_MDL` lookup table.
 ///
 /// The methods implemented on `Of` always return a valid value.
-#[derive(PartialEq, PartialOrd, Copy, Clone)]
+#[derive(PartialEq, PartialOrd, Copy, Clone, Ord, Eq)]
```

Comment:
From one point of view, yes, if it is private, that it will not help users. From the other point - it is likely, that it will be required for something inside `chrono` and therefore will be added anyway.

In any case, I can remove this derives for private types, if you want. I just think, that it is better, then even private types have all the applicable traits derived.

Reply:
Derives aren't free, exactly -- they generate a bunch of code which all downstream users will have to compile, adding to compile times. Plus, I'd rather we add them when we need them rather than proactively -- I feel like the use of `Eq` and `Ord` is pretty rare and we're unlikely to need them for these types.

Reply:
Fair enough, I removed them.